src/backend/TrafficCourts/Workflow.Service/Services/EmailSenderService.cs

```diff
@@ -162,47 +151,74 @@ public async Task SendEmailAsync(SendEmail emailMessage, CancellationToken cance
                 _logger.LogError(scfe, "An error connecting to the the SMTP Server");
                 throw new EmailSendFailedException("An error connecting to the the SMTP Server", scfe);
             }
+            finally
+            {
+                await SaveEmailtoHistory(emailMessage, sentSuccessfully);

```

Comment:
Should this be in it's own try-catch?  If this errors, then the error will propagate to the calling method. The initial email could have been send successfully and this saving to history fail.

Reply:
will do